Helper/TbkResponseHelper.php

```diff
@@ -0,0 +1,197 @@
+        try {
+            $scopeConfig = ObjectManagerHelper::get(ScopeConfigInterface::class);
+            $timezone = $scopeConfig->getValue('general/locale/timezone');
+
+            $utcDate = new DateTime($utcDate, new DateTimeZone('UTC'));
+            $utcDate->setTimezone(new DateTimeZone($timezone));
+
+            return $utcDate->format('d-m-Y H:i:s P');
+        }
+        catch (\Exception $e) {
+            return $e;
+        }
+    }
+
+    /**
+     * Retrieve success message for a transaction
+     *
+     * @param object|array $transactionResult
+     * @param string $product
+     * @param $paymentTypeCodearray
+     * @return string success message
+     */
+    public static function getSuccessMessage($transactionResult, string $product)
+    {
+
+        if (strpos($product, 'click')) {
+            $transactionResult = self::getOneclickDetails($transactionResult);
+        }
+
+        if ( in_array($transactionResult->paymentTypeCode, ['SI', 'S2', 'NC', 'VC']) ) {
+            $tipoCuotas = self::$paymentTypeCodearray[$transactionResult->paymentTypeCode];
+        } else {
+            $tipoCuotas = 'Sin cuotas';
+        }
+
+        if ($transactionResult->responseCode == 0) {
+            $transactionResponse = 'Transacci&oacute;n Aprobada';
+        } else {
+            $transactionResponse = 'Transacci&oacute;n Rechazada';
+        }
+
+        if ($transactionResult->paymentTypeCode == 'VD') {
+            $paymentType = 'Débito';
+        } elseif ($transactionResult->paymentTypeCode == 'VP') {
+            $paymentType = 'Prepago';
+        } else {
+            $paymentType = 'Crédito';
+        }
+        $installmentsString = '';
+        if ($tipoCuotas != 'Sin cuotas') {
+            $installmentsString = "
+                <div>
+                    • N&uacute;mero de cuotas: <b>{$transactionResult->installmentsNumber}</b>
+                </div>
+                <div>
+                    • Monto Cuota: <b>{$transactionResult->installmentsAmount}</b>
+                </div>
+            ";
+        }
+
+        return "
+            <b>Detalles del pago con {$product}</b>
+            <div>
+                • Respuesta de la Transacci&oacute;n: <b>{$transactionResponse}</b>
+            </div>
+            <div>
+                • C&oacute;digo de la Transacci&oacute;n: <b>{$transactionResult->responseCode}</b>
+            </div>
+            <div>
+                • Monto: <b>$ {$transactionResult->amount}</b>
+            </div>
+            <div>
+                • Order de Compra: <b>$ {$transactionResult->buyOrder}</b>
+            </div>
+            <div>
+                • Fecha de la Transacci&oacute;n: <b>".substr(self::utcToLocalDate($transactionResult->transactionDate), 0,10).'</b>
+            </div>
+            <div>
+                • Hora de la Transacci&oacute;n: <b>'.substr(self::utcToLocalDate($transactionResult->transactionDate),11, 8)."</b>
```

Comment:
you can make some specific method to get hour in the necessary format instead use substring